src/app/home/todo-card/todo-card.component.scss

```diff
@@ -5,6 +5,19 @@
   background: #fff;
   min-height: 58px;
 }
+.todo-item {
```

Comment:
This class has been removed already. Do we still need this CSS ?

Reply:
I use this to limit long description text in the todo items on dashboard.

Reply:
it was there i didn't add this. I have added this part.
`.icon-item {
    ion-text {
      p {
        max-height: 16px;
        -webkit-line-clamp: 1;
        line-height: 16px;
        overflow: hidden;
      }
    }
  }`